index.html

```diff
@@ -0,0 +1,475 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Space Ripples Reveal Big Bang's Smoking Gun - The New York Times</title>
+        <link href="https://fonts.googleapis.com/css2?family=Libre+Franklin&display=swap" rel="stylesheet" />
+        <link href="https://fonts.googleapis.com/css2?family=Domine&display=swap" rel="stylesheet" /> 
+        <script src="https://kit.fontawesome.com/5036f34c07.js" crossorigin="anonymous"></script>
+    </head>
+
+    <!-- navbar section -->
+
+<body class="m-0 p-0">
+    <header class="m-0 p-0">
+        <nav class="navbar dflex z-index-top">
+            <div class="nav-icon-left dflex">
+                <i class="fas fa-bars mr"></i> 
+                <i class="fas fa-search"></i>
+                <a href="#" class="black"><strong>SPACE & COSMOS</strong></a>
+            </div>
+            <div>
+                <a class="black"  href="#"><img src="images/logo.png" alt="New York Times Logo"></a>
+            </div>
+            <div class="sub-btn dflex">
+                <button type="button" class="nav-btn-1 nav-btn">SUBSRCIBE NOW</button>
+                <button type="button" class="nav-btn">LOG IN</button>
+            </div>
+        </nav>
+    </header>
+
+    <main>
+            <!-- Main Article section -->
+
+        <section class="article-main">
+
+            <h1 class="article-titel font-style">Space Ripples Reveal Big Bang’s <br>
+            Smoking Gun</h1>
+
+            <div class="centered">
+                <figure class="m-bot-0">
+                    <img class="article-pic m-bot-0" src="images/18cosmos-jumbo.jpg" alt="picture of the cosmos">
+                </figure>
+                <div class="figcap-1 m-top-0">
+                    <p class="m-bot-0">Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p> <small class="m-top-0 figcap-small">Rick Friedman for The New York Times</small>
+                </div>
+            </div>
+
+            
```

Comment:
- [OPTIONAL] I suggest you to use a code-formatters to avoid redundant indentation and  empty lines.